Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -328,12 +330,22 @@ class SettingsTableViewController: UITableViewController {
     override func tableView(tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
         let headerView = tableView.dequeueReusableHeaderFooterViewWithIdentifier(SectionHeaderIdentifier) as! SettingsTableSectionHeaderFooterView
         let sectionSetting = settings[section]
+
         if let sectionTitle = sectionSetting.title?.string {
             headerView.titleLabel.text = sectionTitle
+            if sectionTitle == Bug1209692 {
+                headerView.titleAlignment = .Top
+                headerView.titleLabel.numberOfLines = 0
+                headerView.showBottomBorder = false
+
+                headerView.titleLabel.snp_updateConstraints { make in
+                    make.right.equalTo(headerView).offset(-70)
+                }
+            }
         }
 
         // Hide the top border for the top section to avoid having a double line at the top
-        if section == 0 {
+        if section == 0 || (!profile.hasAccount() && section == 2) {
```

Comment:
I would move this into AppSettingsViewController and override this method instead of having it here because subclasses probably don't want this functionality.